cypress/fixtures/articles_index.json

```diff
@@ -0,0 +1,12 @@
+{
+  "articles": [
+    {
```

Comment:
include real formate of response json pls

Reply:
Done